pwndbg/commands/windbg.py

```diff
@@ -78,7 +78,7 @@ def dX(size, address, count, to_string=False):
     values = []
     address = int(address) & pwndbg.arch.ptrmask
     type   = get_type(size)
-    for i in range(count):
+    for i in range(int(count)):
```

Comment:
We use `count` directly below.  It's probably best to do `count = int(count)` above, rather than inline here.